CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.11.1 - 2019-10-16
+### Added
+- Update mongodb package to 3.3.2
```

Comment:
As discussed offline - would like to know what functional changes this has for the user. Right now it just sounds like engineering work, which we don't typically put in the changelog

Reply:
@fiveisprime Could we discuss what exactly updating the mongodb package fixed for the customer either offline or in a comment in this thread?